metricbeat/module/elasticsearch/pending_tasks/_meta/data.json

```diff
@@ -1,24 +1,26 @@
 {
-  "@timestamp": "2018-05-03T20:58:17.379Z",
-  "metricset": {
-    "rtt": 2487,
-    "namespace": "elasticsearch.cluster.pending_task",
-    "name": "pending_tasks",
-    "module": "elasticsearch",
-    "host": "localhost:9200"
-  },
-  "elasticsearch": {
-    "cluster": {
-      "pending_task": {
-        "insert_order": 47,
-        "priority": "HIGH",
-        "source": "put-mapping",
-        "time_in_queue.ms": 34
-      }
+    "@timestamp": "2018-05-03T20:58:17.379Z",
+    "metricset": {
+        "rtt": 2487,
+        "namespace": "elasticsearch.cluster.pending_task",
+        "name": "pending_tasks",
+        "module": "elasticsearch",
+        "host": "localhost:9200"
+    },
+    "elasticsearch": {
+        "cluster": {
+            "id": "3LbUkLkURz--FR-YO0wLNA",
+            "name": "es1",
+            "pending_task": {
+                "insert_order": 47,
+                "priority": "HIGH",
+                "source": "put-mapping",
+                "time_in_queue.ms": 34
+            }
+        }
+    },
+    "agent": {
```

Comment:
I don't think this part is correct for 6.x. Missed it in the other backport PR too.

Reply:
Ahhh, good catch! I'll re-run the integration test to generate data in each 6.x PR and hopefully it will fix it. Thanks!